lib/blocks.php

```diff
@@ -292,7 +295,7 @@ function strip_dynamic_blocks( $content ) {
 	 * @return string
 	 */
 	function strip_dynamic_blocks_add_filter( $text ) {
-		add_filter( 'the_content', 'strip_dynamic_blocks', 6 ); // Before do_blocks().
+		add_filter( 'the_content', 'strip_dynamic_blocks', $the_content_priority_strip_dynamic_blocks );
```

Comment:
@youknowriad for some reason I'm getting `Undefined variable: the_content_priority_strip_dynamic_blocks` here.

Scoping issues?

Reply:
🤔

Reply:
yes, too much JS in my head. it should be fixed now.

Reply:
If we're `global`-ing already here + lengthy-named tracker variables across 2 files, is there some decent global class property available instead?

`GutenbergVariables::$the_content_priority['strip_dynamic_block'] = 6`, etc.

Reply:
In the wild, I'm seeing plugins that need to do special content processing, currently have to hardcode things like `remove_filter the_content gutenberg_wpautop 8`, which means they instantly break when a release comes along that changes priorities.

Which is why some kind of a "well-known things" registry to pull from, would be much nicer for everybody.

But I understand this is a massive undertaking and can't be addressed here.

Reply:
@lkraav fair point and I'd suggest opening a trac issue for it as, this code is already merged there for 5.0 beta.

Reply:
I restored the hard-coded priorities for now. The variables were messing with the php unit tests. Any refactoring should happen separately.